style.css

```diff
@@ -0,0 +1,124 @@
+* {
+    box-sizing: border-box;
+}
+
+main {
+    height: 768px;
+    width: 1024px;
+    border: solid black 5px;
+    background: indianred url("images_folder/halloweenBg.png") no-repeat;
+    background-size: cover;
+    background-position: center;
+    opacity: 0.8;
+}
+
+#topSection {
+    height: 70px;
+    padding: 15px;
+    display: block;
+}
+
+#title {
+    width: 300px;
+    margin: 0 auto;
+    border: white solid 5px;
+    text-align: center;
+}
+
+#gameSection {
+     height: 600px;
+
+ }
+
+#button {
+    height: 50px;
+    width: 100px;
+    text-align: center;
+    padding: 5px;
+    margin-top: -400px;
+    margin-right: auto;
+    margin-left: auto;
+    border: solid white 5px;
+    display: block;
+    color:white;
+}
+
+#gameScreen {
+    width: 900px;
+    height: 660px;
+    margin: 0 auto;
+    background: url("images_folder/map_designV2.png") no-repeat;
+    background-size: cover;
+    background-position: center;
+    visibility: hidden;
+    z-index: -1;
+}
+
+.Height30percent {
+    height: 30%;
+}
+
+#rightLeg {
+    float: left;
+    height: 10%;
+    width: 5%;
+    background:  url("images_folder/reaper_sprite_right.png") no-repeat;
+    background-size: cover;
+    background-position: center;
+    margin-top: 280px;
+    margin-left: 10px;
+    position: absolute;
+}
+
+#leftLeg {
+    float: left;
+    height: 10%;
+    width: 5%;
+    background:  url("images_folder/reaper_sprite_right.png") no-repeat;
+    background-size: cover;
+    background-position: center;
+    margin-top: 280px;
```

Comment:
Margin-top 280 may well play with the page in ways that you don't want it to.. just a suggestion to use something else.. up to you.